install.cmd

```diff
@@ -0,0 +1,3 @@
+@set PATH=%ProgramFiles(x86)%\Microsoft Visual Studio 14.0\Common7\IDE;%PATH%
+vsixinstaller /q /u:"c3d3dc68-c977-411f-b3e8-03b0dccf7dfc"
+vsixinstaller "%cd%\build\Debug\GitHub.VisualStudio.vsix"
```

Comment:
Nevermind the bit about installing to 2017, the vsix installer does that automatically. The question is, though, if you run the uninstaller in quiet mode, does it uninstall from all versions? My guess is probably yes (I think the uninstaller selects everything by default)